include/velocypack/Iterator.h

```diff
@@ -346,7 +346,7 @@ class ObjectIterator {
     if (head == 0x14) {
       return _slice.start() + _slice.getStartOffsetFromCompact();
     } else if (useSequentialIteration) {
-      return _slice.begin() + _slice.findDataOffset(head);
+      return _slice.start() + _slice.findDataOffset(head);
```

Comment:
Why Slice have to different names method but both makes same?

Reply:
Can you make same for ArrayIterator?

Reply:
Good question. I don't know, but both are totally equivalent.